package-lock.json

```diff
@@ -2,22 +2,18 @@
     "name": "universal-runner",
     "version": "1.0.0",
     "lockfileVersion": 1,
-    "requires": true,
     "dependencies": {
         "acorn": {
-            "version": "5.1.1",
-            "resolved": "https://registry.npmjs.org/acorn/-/acorn-5.1.1.tgz",
-            "integrity": "sha512-vOk6uEMctu0vQrvuSqFdJyqj1Q0S5VTDL79qtjo+DhRr+1mmaD+tluFSCZqhvi/JUhXSzoZN2BhtstaPEeE8cw==",
+            "version": "5.1.2",
```

Comment:
@tsvetomir-nedyalkov Should this `-lock.json` file be in source control?

Reply:
@tejasranade Yes, it should be commited - the benefits are described here:
https://docs.npmjs.com/files/package-lock.json